releng/make-commit-log.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+cleanup() {
+  [ -f "${MASTER_COMMITS}" ] && rm "${MASTER_COMMITS}"
+  [ -f "${RELEASE_COMMITS}" ] && rm "${RELEASE_COMMITS}"
+  git checkout "${CUR_BRANCH}"
```

Comment:
```suggestion
  [ -n "${CUR_BRANCH}" ] && git checkout "${CUR_BRANCH}"
```

Reply:
Probably more error checking than this script deserves, but added